BraintreeCore/src/main/java/com/braintreepayments/api/AnalyticsDatabase.java

```diff
@@ -151,7 +151,10 @@ public void run() {
                         request.id = Integer.valueOf(ids[i]);
                         request.event = events[i];
                         request.timestamp = Long.valueOf(timestamps[i]);
-                        request.metadata = new JSONObject(cursor.getString(cursor.getColumnIndex(META_JSON)));
+                        int index = cursor.getColumnIndex(META_JSON);
+                        if (index >= 0) {
+                            request.metadata = new JSONObject(cursor.getString(index));
+                        }
```

Comment:
This fixes a compiler lint error when running the tests from Android Studio Arctic Fox (required for Android 12 emulators)